docs/core-study-details.md

```diff
@@ -368,6 +400,7 @@ making sure to include all associated resources.
 |   Column   | Type  |Description|
 |------------|-------|-----------|
 |id          |varchar|           |
+|row         |bigint |           |
```

Comment:
yeah I actually made some updates in one of the other PRs, i should regen this anyway.

Reply:
i am still seeing this after regen.

I've been running this against a synthetic dataset, which might have different structure? and/or may be out of date?

I'm going to leave this as is for now rather than rebuilding the cerner table before we release breaking changes, but I'm going to add a staging DB at some point that we can use for stuff like this.